projects/storefrontapp-e2e-cypress/cypress/helpers/cart-import-export.ts

```diff
@@ -286,7 +302,6 @@ export function attemptUpload(csvPath: string) {
     'cx-import-entries-dialog textarea[formcontrolname="description"]'
   ).type('A test description.');
 
-  cy.intercept('GET', '**/users/current/carts/*?**').as('import');
```

Comment:
I removed this and do not include a cy.wait() for this after the import button is clicked because we have test cases where the import should fail. That means we will never get the saved cart and the cy.wait() would fail.

Reply:
let's talk about this as we might want to check if the product that tried to be added failed and/or have the cart - refreshed after failing an upload

Reply:
nvm